src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/AtomicReshaper.java

```diff
@@ -0,0 +1,179 @@
+package com.cleanroommc.groovyscript.compat.mods.prodigytech;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.ingredient.OreDictIngredient;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import lykrast.prodigytech.common.recipe.AtomicReshaperManager;
+import lykrast.prodigytech.common.util.Config;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@RegistryDescription
+public class AtomicReshaper extends VirtualizedRegistry<AtomicReshaperManager.AtomicReshaperRecipe> {
+    @RecipeBuilderDescription(example = {
+        @Example(".input(item('minecraft:gold_ingot')).output(item('minecraft:emerald_block')).primordium(10).time(50)"),
+        @Example(".input(item('minecraft:gold_block')).output(item('minecraft:diamond_block'), 10).output(item('minecraft:carrot'), 3).primordium(7)")
+    })
+    public AtomicReshaper.RecipeBuilder recipeBuilder() {

```

Comment:
since `RecipeBuilder` is the only thing called that accessible in this file, theres no need to include `AtomicReshaper.` before it, in any of the places its used here. this also applies to the other files. not a big deal, and honestly doesnt need to get changed, but thought i should note it. 
why more word when few word do trick?